tests/workflows/basic_tool_calling_node/workflow.py

```diff
@@ -0,0 +1,84 @@
+from vellum.client.types.chat_message import ChatMessage
+from vellum.client.types.chat_message_prompt_block import ChatMessagePromptBlock
+from vellum.client.types.function_definition import FunctionDefinition
+from vellum.client.types.plain_text_prompt_block import PlainTextPromptBlock
+from vellum.client.types.rich_text_prompt_block import RichTextPromptBlock
+from vellum.client.types.variable_prompt_block import VariablePromptBlock
+from vellum.workflows.nodes.experimental.tool_calling_node import ToolCallingNode
+from vellum.workflows.workflows.base import BaseWorkflow
+
+
+def get_current_weather(location: str, unit: str) -> str:
+    """
+    Get the current weather in a given location.
+    """
+    return f"The current weather in {location} is sunny with a temperature of 70 degrees {unit}."
+
+
+class GetCurrentWeatherNode(ToolCallingNode):
+    """
+    A tool calling node that calls the get_current_weather function.
+    """
+
+    ml_model = "gpt-4o-mini"
+    blocks = [
+        ChatMessagePromptBlock(
+            chat_role="SYSTEM",
+            blocks=[
+                RichTextPromptBlock(
+                    blocks=[
+                        PlainTextPromptBlock(
+                            text="You are a weather expert",
+                        ),
+                    ],
+                ),
+            ],
+        ),
+        ChatMessagePromptBlock(
+            chat_role="USER",
+            blocks=[
+                RichTextPromptBlock(
+                    blocks=[
+                        VariablePromptBlock(
+                            input_variable="question",
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    ]
+    function_callables = {"get_current_weather": get_current_weather}
```

Comment:
[oos] we shouldn't need this attribute -  `functions` should already support callables

Reply:
we also shouldn't need the `FunctionDefinition` below. Should be as simple as:

`functions = [get_current_weather]`